services/dynamic-sidecar/tests/conftest.py

```diff
@@ -278,3 +280,30 @@ def mock_core_rabbitmq(mocker: MockerFixture) -> dict[str, AsyncMock]:
             autospec=True,
         ),
     }
+
+
+@pytest.fixture
+def mock_stop_heart_beat_task(mocker: MockerFixture) -> AsyncMock:
+    return mocker.patch(
+        "simcore_service_dynamic_sidecar.modules.resource_tracking.core.stop_heart_beat_task",
+        return_value=None,
+    )
+
+
+@pytest.fixture
+def mock_metrics_params(
+    mock_stop_heart_beat_task: AsyncMock,
+) -> CreateServiceMetricsAdditionalParams:
+    return CreateServiceMetricsAdditionalParams(
+        wallet_id=1,
+        wallet_name="test_wallet",
+        product_name="test",
+        simcore_user_agent="",
+        user_email="",
```

Comment:
you could use the faker for all of these. or an example in the ```CreateServiceMetricsAdditionalParams``` model. Anyhow I think having empty values might make you miss some test cases.